### PR TITLE
Use `GENERAL_SHOW_TIME` for decryption screen timeout

### LINKED ISSUE #1973: [FEATURE] add password visibility-timeout setting
### Is your feature request related to a problem? Please describe.

The decrypted password will be shown for ~45 seconds before the UI goes automatically back to the password list.

The 45s timeout is too short for me when I'm writing the password to another device (especially for long random auto-generated password)

### Describe the solution you'd like

An option could be added in settings to define a custom password timeout

### Describe alternatives you've considered

The value of "Password copy timeout" could instead be used as settings for visibility timeout rather than having two settings

Since the setting is called GENERAL_SHOW_TIME it would make sence to combine the two

### Additional context

I'm currently using version 1.13.5, and tryed to change "Password copy timeout" to a higher value but password visibility stayed at 45sec

Thanks for the great App BTW.

### PR DESCRIPTION
**Pull request by Skrilltrax**
Instead of adding another option, this uses the alternate approach mentioned in #1973. Let me know if any changes are required.

### CODE REVIEW

File: app/src/main/java/app/passwordstore/ui/crypto/DecryptActivity.kt [hunk 1]
```diff
@@ -91,13 +92,16 @@ class DecryptActivity : BasePgpActivity() {
...
-   * Automatically finishes the activity 60 seconds after decryption succeeded to prevent
-   * information leaks from stale activities.
+   * Automatically finishes the activity after [PreferenceKeys.GENERAL_SHOW_TIME] seconds decryption
+   * succeeded to prevent information leaks from stale activities.
    */
   private fun startAutoDismissTimer() {
     lifecycleScope.launch {
-      delay(60.seconds)
-      finish()
+      val timeout = settings.getString(PreferenceKeys.GENERAL_SHOW_TIME)?.toIntOrNull() ?: 45
```
Comment (msfjarvis) [inline app/src/main/java/app/passwordstore/ui/crypto/DecryptActivity.kt]: This needs to be 60 and pulled into a const.

**Review by msfjarvis — CHANGES_REQUESTED** [review]
Once the changes are made, run the `detektBaseline` task and commit the changes.

File: app/src/main/java/app/passwordstore/ui/crypto/DecryptActivity.kt (hunk 1 above)
Comment (Skrilltrax) [reply app/src/main/java/app/passwordstore/ui/crypto/DecryptActivity.kt]: The default value for `GENERAL_SHOW_TIME` is 45 at other places so I kept it 45 here as well.

Comment (msfjarvis) [reply app/src/main/java/app/passwordstore/ui/crypto/DecryptActivity.kt]: Then raise that to 60 as well.

**Comment by msfjarvis** [comment]
Rebase and run Spotless + Detekt

**Review by msfjarvis — APPROVED** [review]

**Pull request merged**